src/commands/pe-symbols/README.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+
+Upload Windows PE debug info files to Datadog to symbolicate your profiles.
+
+
+## Setup
+
+You need to have `DATADOG_API_KEY` in your environment.
+
+```bash
+# Environment setup
+export DATADOG_API_KEY="<API KEY>"
+```
+
+You can configure the tool to use Datadog EU by defining the `DATADOG_SITE` environment variable as `datadoghq.eu`. By default, the requests are sent to Datadog US.
+
+To make these variables available, Datadog recommends setting them in an encrypted `datadog-ci.json` file at the root of your project:
+
+```json
+{
+  "apiKey": "<DATADOG_API_KEY>",
+  "datadogSite": "<DATADOG_SITE>"
+}
+```
+
+To override the full URL for the intake endpoint, define the `DATADOG_SOURCEMAP_INTAKE_URL` environment variable.
+
+## Commands
+
+### `upload`
+
+**Warning:** The `pe-symbols upload` command is in beta. It requires you to set `DD_BETA_COMMANDS_ENABLED=1`.
+
+This command will upload debug info from Windows PE files to Datadog in order to symbolicate your application's profiles.
```

Comment:
```suggestion
This command uploads debug info from Windows PE files to Datadog in order to symbolicate your application's profiles.
```
Sorry, I missed this on the first pass. (The style guide for the Datadog public docs asks that we use present tense always.)